src/mongo/db/storage/wiredtiger/wiredtiger_record_store.h

```diff
@@ -277,9 +287,9 @@ class WiredTigerRecordStore : public RecordStore {
     const bool _isEphemeral;
     // True if the namespace of this record store starts with "local.oplog.", and false otherwise.
     const bool _isOplog;
-    const int64_t _cappedMaxSize;
+    int64_t _cappedMaxSize;
     const int64_t _cappedMaxSizeSlack;  // when to start applying backpressure
-    const int64_t _cappedMaxDocs;
```

Comment:
I think it a good idea of changing the two variables in order to support capped-set online change size.
But wt's mongo layer uses OplogStones to track whether it is necessary to truncate oplogs from the beginning.
_oplogStones is initialized once the service is started and never ever changed unless a manually truncate is applied to it. I didnt see any changes to _oplogStones in this patch, maybe I'm wrong?